crates/grafbase-local-backend/src/dev/configurations.rs

```diff
@@ -82,6 +82,8 @@ pub async fn get_and_merge_configurations(
         .map(|config| config.subgraphs.into_keys().collect::<HashSet<_>>())
         .unwrap_or_default();
 
+    merged_configuration.graph.introspection = true;
```

Comment:
Since that might be confusing, could we emit a warning saying we force enable introspection in cases where introspection was explicitly disabled in the configuration?

Reply:
Yep.